janus_server/src/bin/janus_cli.rs

```diff
@@ -101,6 +101,10 @@ enum Command {
 
 impl Command {
     async fn execute(&self) -> Result<()> {
+        let kube_client = kube::Client::try_default()
+            .await
+            .context("couldn't connect to Kubernetes environment")?;
+
```

Comment:
We need to defer this, and only run it inside the commands that require it. Otherwise, we may get the following error when it's not necessary.

```
Error: couldn't connect to Kubernetes environment

Caused by:
    0: Failed to infer configuration: failed to infer config: in-cluster: (failed to read the default namespace: No such file or directory (os error 2)), kubeconfig: (failed to read kubeconfig from '"/root/.kube/config"': No such file or directory (os error 2))
    1: failed to infer config: in-cluster: (failed to read the default namespace: No such file or directory (os error 2)), kubeconfig: (failed to read kubeconfig from '"/root/.kube/config"': No such file or directory (os error 2))
    2: failed to read kubeconfig from '"/root/.kube/config"': No such file or directory (os error 2)
    3: No such file or directory (os error 2)
```

Reply:
Whoops, I was sure I had caught this, but I think I got mixed up while rebasing.